portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,64 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Alexandre Lopes</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
+      <div class="header">
+        <nav>
+            <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#about">About</a></li>
+            </ul>
+        </nav>
+      </div>
+
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <section class="home" id="home">
+        <div class="max-width">
+            <div class="home-content">
+                <div class="text-1">
+                    Hello World!
+                </div>
+                <div class="text-2">
+                    I'm Alexandre
+                </div>
+                <div class="text-3">
+                    Undergrad CS student
+                </div>
+                <a href="./images/AlexandreLopes_Resume.pdf" target="_blank" download" class="button-home">Downlaod Resume</a>
+            </div>
+        </div>
+    </section>
+    <div class="container" id="about">
+            <header>
+                <h1>About</h1>
+            </header>
+                <div class="about-content">
+                    <p>Hello! My name is Alexandre Miguel. I am a college student (3.91 GPA, Bachelor of Science
+                        in Computer Science) currently attending the University of South Florida. For this Spring
+                        of 2021, I'm joining Google in an 11-week program created for college students interest
+                        in building new tools and connecting with the tech community. We will be learning web
+                        applications tools such as java, HTML, CSS and JavaScript. In addition to that, we will conduct
+                        code reviews and learn industry-aligned skills. I am so excited to learning skills and values
+                        from a company like Google that will help me prepare for my future career.
+                    </p>
+                    <p>
+                        Some Technologies I've worked with include:
+                    </p>
+                    <ul>
+                        <li>Java</li>
+                        <li>C</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                    <!--Adding a servlet to portofolio-->
+                    <p>
+                         <a href="/String">Click here to access servlet</a>
+                    </p>
+                </div>
     </div>
```

Comment:
The indentation in this area is still kind of odd, but I'm okay with you fixing it in a later commit

Reply:
Do you have any tips to make the indentation looks better? Thank you for the feedback!

Reply:
In general, child elements should be one indentation level further than their parent element, and sibling elements should be at the same indentation level. In this section, the <header> element is a child of the div with the `container` class. You seem to be using 4-space indentation, so the <header> and </header> tags should be indented 4 spaces further than that <div> instead of 8. The div with the `about-content` class is a sibling of the header element, so they should be at the same indentation level